single_tech_samples/databricks/sample4_ci_cd/README.md

```diff
@@ -166,53 +273,128 @@ cd ..
 rm -rf modern-data-warehouse-dataops
 
 git init
-git remote add origin <your repo url>
+git remote add origin [your repo url]
 git add -A
 git commit -m "first commit"
 git push -u origin --all
+git branch develop master
+git branch staging develop
+git branch production staging
 ```
 
-1. In your Azure DevOps project
-    - create a ci pipeline and select ci-pipeline.yml. it will run unit tests and publish the artifacts when changes committed.
-    - create a cd pipeline and select cd-pipeline.yml. it will run integration tests when ci pipeline successfully run and the go-production deployment can be manually triggered.
+After running the scripts, you can open the your repo url to check the code is pushed to the repository.
+
+There are 3 branch in the repository:
+
+- **develop branch** is the code base of development
+- **staging branch** is for intergation testing
+- **production branch** is for production deployment
+
+You can find the [document](https://docs.microsoft.com/en-us/azure/devops/repos/git/branch-policies?view=azure-devops) to set branch policy.
+
+#### 2.4.2 DevOps pipeline setup
+
+In this repo, there are several yml files which are the pipelines to support the CI/CD. you need to import the yml as build pipeline.
+
+- Import [./doveops/notebook-pipelines.yml](./devops/notebook-pipelines.yml) as build pipeline. This pipeline tests and uploads the notebooks to databricks workspace.
```

Comment:
nit: typo, doveops.

This is also in other places -- copy paste error? :)

Reply:
fix in [this commit](https://github.com/Azure-Samples/modern-data-warehouse-dataops/pull/359/commits/431a918f38a48c319a7a146c29bbf129fe870b46)
Thanks!